build.gradle.kts

```diff
@@ -73,4 +65,16 @@ subprojects {
       }
     }
   }
+
+  apply(plugin = "com.diffplug.spotless")
+  pluginManager.withPlugin("com.diffplug.spotless") {
```

Comment:
You don't need this if you're applying it right above

Reply:
Nice nice, will change